pom.xml

```diff
@@ -152,7 +152,7 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>2.9.10.1</version>
+            <version>2.9.10.5</version>
```

Comment:
Since you are compiling against 5.0 anyway, please use the same version as 5.0 is using - namely, 2.11.1
And you should use the same version for all jackson dependencies.
Also, even though such dependencies are not included in jar file automatically, having <scope>provided</scope> provides better visibility that they will be provided by Symphony.
Finally, can you please check other dependencies versions and make sure they match the ones used by Symphony (versions declared there in pom.xml of symphony-parent project) - I see few more mismatches.